python/fio/README.md

```diff
@@ -0,0 +1,43 @@
+# arca-fio
+
+This plugin executes the fio workload software with a given input schema defined in a yaml file.
+
+## Run the thing
+
+### Containerized
+
+#### Requirements
+
+* fio_plugin image named `fio-plugin`
+* current working directory is the fio plugin root
+
+```shell
+cat fixtures/poisson-rate-submission_input.yaml | docker run fio-plugin -f -
```

Comment:
You have both `fixtures` and `mocks` subdirs in the repo with separate references in the README and the code.